config/action.d/nftables-allports.conf

```diff
@@ -17,6 +17,6 @@ before = nftables-common.conf
 # Notes.:  additional expressions for nftables filter rule
 # Values:  nftables expressions
 #
-nftables_mode = ip protocol <protocol>
+nftables_mode = meta l4proto <protocol>
```

Comment:
> change `ip protocol` to `meta l4proto` in `nftables-allports.conf` since the former only works for IPv4

I don't think so (ATM I don't have nft to hand to check), just try following:
```INI
nftables_mode = ip protocol <protocol>
[Init?family=inet6]
nftables_mode = ip6 protocol <protocol>
```

Reply:
Or much better, because you already have parameter `address_family`, just try to rewrite (without new section):
```diff
-nftables_mode = ip protocol <protocol>
+nftables_mode = <address_family> protocol <protocol>
```

Reply:
Yeah I thought it would work but it doesn't (I wouldn't have changed the code otherwise).
IPv6 doesn't have a protocol header so `ip6 protocol` makes no sense. See [the reply I got on the netfilter ML](https://marc.info/?l=netfilter&m=150519799600765&w=2).

Reply:
Understood